compression/2.2/modules/ROOT/pages/compression-module-archive-extract.adoc

```diff
@@ -0,0 +1,152 @@
+= Archiving and Extracting Multiple Entries
+
+The Compression module provides the *Archive* operation that enables you to compress a set of entries into a new archive file and the *Extract* operation that enables you to decompress content that represents an archive file. 
+
+
+== Configure Archive Operation
+
+Given an input stream payload, the *Archive* operation enables you to compress all the payload entries into a new archive file in the `Zip` format. The operation receives a map that identifies the entries' values to compress. Then, the operation receives each entry, which is placed inside the compressed archive file with the entry name you provide in a DataWeave expression.
+
+. In Studio, drag the *Archive* operation to your flow.
+. Set *Entries* to a DataWeave script defining each name of the entry to compress as a key and the content of that entry as its value, for example:
++
+[source,dataweave, linenums]
+----
+{
+         summary.pdf: vars.summary,
+         'details/result_001.pdf': vars.file1
+         'details/result_002.pdf': vars.file2
+       }
+----
+
+
+image::compression-archive-operation.png[Compressor operation configuration]
+
+In the *Configuration XML* editor, the configuration looks like this:
+
+[source, xml, linenums]
+----
+<compression:archive>
+   <compression:entries>
+    #[
+       {
+         summary.pdf: vars.summary,
+         'details/result_001.pdf': vars.file1
+         'details/result_002.pdf': vars.file2
+       }
+     ]
+   </compression:entries>
+   <compression:archiver>
+       <compression:zip-archiver/>
+   </compression:archiver>
+</compression:archive>
+----
+
+The resulting archive contains three entries, one named `summary.pdf` at the root level, the others called `result_001.pdf` and `result_002.pdf`, inside a directory called `details`:
+
+[source]
+----
++- content.zip
+|  \- summary.pdf
+|  \+ details
+   |  \- result_001.pdf
+   |  \- result_002.pdf
+
+----
+
+Note that the slash `/` in the name of an entry, for example, `details/result_001.pdf` indicates a directory separation. All names are introspected to create directories inside the archive.
+
+[NOTE]
+If you create the input to the *Archive* operation using a DataWeave expression, the DataWeave expression must output an object, used to build the Java map object. Define unique keys to add the entire object to the archive.
+
+== Configure ZIP64 Archiver Strategy
+
+An archiver strategy compresses multiple entries into an archive. The Zip archiver strategy declares that the content must be compressed using the `Zip` format. In Studio, the *Archiver* field is set by default. To archive files and byte arrays greater than 4 GB, select the *Force ZIP64* field.
```

Comment:
I missed this earlier, but does I'm not sure "Zip format" needs be marked in backticks